python/python/Ice/ToStringMode.py

```diff
@@ -10,16 +10,19 @@ class ToStringMode(EnumBase):
 
     Enumerators:
```

Comment:
Made it consistent with the generated code. There's probably other hand-written enums that need fixing too.
I stopped after 2 because I was using these as test dummies while figuring out how to make enums render better.